cime_config/config_grids.xml

```diff
@@ -6107,25 +6107,31 @@
     <gridmap glc_grid="mpas.gis4to40km" ocn_grid="oQU240wLI">
       <map name="OCN2GLC_SHELF_FMAPNAME">cpl/gridmaps/oQU240wLI/map_oQU240wLI_to_gis4to40_esmfaave.20250303.nc</map>
       <map name="OCN2GLC_SHELF_SMAPNAME">cpl/gridmaps/oQU240wLI/map_oQU240wLI_to_gis4to40_esmfbilin.20250303.nc</map>
-      <map name="OCN2GLC_TF_SMAPNAME">cpl/gridmaps/oQU240wLI/map_oQU240wLI_to_gis4to40_esmfaave.20250303.nc</map>
+      <!-- due to a known issue with ocean extrapolation in MALI, as of 2025-04-04 it is necessary to use a mapping file with
+           extrapolation included for OCN2GLC_TF_SMAPNAME.  After that is fixed, this file should be switched to the
+           version WITHOUT extrapolation:  map_oQU240wLI-nomask_to_gis4to40_esmfbilin.20250403.nc -->
+      <map name="OCN2GLC_TF_SMAPNAME">cpl/gridmaps/oQU240wLI/map_oQU240wLI_to_gis4to40_esmfbilin.20250303.nc</map>
       <map name="GLC2ICE_FMAPNAME">cpl/gridmaps/mpas.gis4to40km/map_gis4to40_to_oQU240wLI_esmfaave.20250303.nc</map>
       <map name="GLC2ICE_SMAPNAME">cpl/gridmaps/mpas.gis4to40km/map_gis4to40_to_oQU240wLI_esmfaave.20250303.nc</map>
       <map name="GLC2OCN_FMAPNAME">cpl/gridmaps/mpas.gis4to40km/map_gis4to40_to_oQU240wLI_esmfaave.20250303.nc</map>
       <map name="GLC2OCN_SMAPNAME">cpl/gridmaps/mpas.gis4to40km/map_gis4to40_to_oQU240wLI_esmfaave.20250303.nc</map>
       <map name="GLC2OCN_LIQ_RMAPNAME">cpl/gridmaps/mpas.gis4to40km/map_gis4to40_to_oQU240wLI_nn.20250303.nc</map>
-      <map name="GLC2OCN_ICE_RMAPNAME">cpl/gridmaps/mpas.gis4to40km/map_gis4to40_to_oQU240wLI_nn.20250303.nc</map>
+      <map name="GLC2OCN_ICE_RMAPNAME">cpl/gridmaps/mpas.gis4to40km/map_gis4to40_to_oQU240wLI-nomask_nn.20250403.nc</map>
```

Comment:
Instead of "nn" should it be "cstmnn" because it was made with a custom tool?

Reply:
Seems reasonable to me.  @jonbob , what do you think?

Reply:
I think that's OK -- it's not generated by the smoothing code where we have generally added that label, as well as the amount of smoothing. But those files were made from an old cess code that was modified to work for us. When we make both files, smoothed and the nearest-neighbor one the smoothing is based on, we've historically left one as "nn" and the other as "rXXXeXXX.cstmnn". But I'm happy to change that if it's preferable

Reply:
Its fine.  Is this code in a repo somewhere?

Reply:
It's a modified version of a cesm/cime tool, but we're working on replacing it instead of tracking it. I'm happy to change the mapping filenames if that better fits the naming convention

Reply:
@rljacob -- I think the names of these files is the only thing holding up this PR. If you prefer "cstmnn" I'm happy to change them

Reply:
No you can leave the names as is.